benefit/data/sheets/CaseField/CaseField-sandl-nonprod.json

```diff
@@ -3,5 +3,8 @@
   {"LiveFrom": "08/03/2022", "CaseTypeID": "Benefit", "ID": "activeHearingVersionNumber", "Label": "Active Hearing Version Number", "FieldType": "Number", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/06/2022", "CaseTypeID": "Benefit", "ID": "overrideFields", "Label": "Overrides Listing Values", "FieldType": "overrideFields", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/06/2022", "CaseTypeID": "Benefit", "ID": "defaultOverrideFields", "Label": "Default Listing Values", "FieldType": "overrideFields", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "06/07/2022", "CaseTypeID": "Benefit", "ID": "amendReasons", "Label": "Amend Reason", "FieldType": "MultiSelectList", "FieldTypeParameter": "FL_amendReason", "SecurityClassification": "PUBLIC"}
+  {"LiveFrom": "06/07/2022", "CaseTypeID": "Benefit", "ID": "amendReasons", "Label": "Amend Reason", "FieldType": "MultiSelectList", "FieldTypeParameter": "FL_amendReason", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "04/07/2022", "CaseTypeID": "Benefit", "ID": "caseOutcome", "Label": "Case Outcome", "FieldType": "FixedList", "FieldTypeParameter": "FL_caseOutcome", "SecurityClassification": "PUBLIC"},
```

Comment:
I just realised that these two need to be in the prod sheet

Reply:
So 6 and 7 need to be in another json file perhaps CaseEventsToFields.json?

Reply:
I have moved CaseOutcome.